packages/fxa-auth-server/lib/subhub/client.js

```diff
@@ -180,22 +180,20 @@ module.exports = function(log, config) {
           orig_system: ORIG_SYSTEM,
         });
       } catch (err) {
-        if (err.statusCode === 400) {
+        if (err.statusCode === 400 || err.statusCode === 402 || err.statusCode === 404) {
```

Comment:
Restating https://github.com/mozilla/fxa/pull/1592#discussion_r296603832, isn't this `if` test redundant now with the tests below? The only side effect would be to call `log.error` in every case, which seems like an improvement to me. Can we delete this outer one?

Reply:
I don't entirely understand what's going on here, but the error message seems to get logged twice nearly identically when I make that change. So... it seems logically sensible to simplify, but I guess I'll leave it be.